UPGRADE-2.0.md

```diff
@@ -3,6 +3,24 @@ UPGRADE FROM 1.x to 2.0
 
  * Classes in the namespace `MongoDB\Operation\` are `final`.
  * All methods in interfaces and classes now define a return type.
+ * The `MongoDB\ChangeStream::CURSOR_NOT_FOUND` constant is now private.
+ * The `MongoDB\Operation\Watch::FULL_DOCUMENT_DEFAULT` constant has been
+   removed.
+ * The `getNamespace` and `isGeoHaystack` methods hav been removed from the
+   `MongoDB\Model\IndexInfo` class.
+ * The `maxScan`, `modifiers`, `oplogReplay`, and `snapshot` options for `find`
+   and `findOne` operations have been removed.
+ * The `MongoDB\Collection::mapReduce` method has been removed. Use aggregation
```

Comment:
Consider linking to https://www.mongodb.com/docs/manual/reference/map-reduce-to-aggregation-pipeline/ if you want to be helpful.

Reply:
Ah, yes, exactly the information people will need. Added a link to this 👍